tendermint/src/validator.rs

```diff
@@ -31,9 +31,9 @@ impl Set {
     }
 
     /// Sort the validators according to the current Tendermint requirements
-    /// (v. 0.33 -> by validator address, ascending)
+    /// (v. 0.34 -> by validator power, descending)
     fn sort_validators(vals: &mut Vec<Info>) {
-        vals.sort_by_key(|v| v.address);
+        vals.sort_by(|a, b| b.voting_power.cmp(&a.voting_power));
```

Comment:
```suggestion
        vals.sort_by_key(|v| std::cmp::Reverse(v.voting_power));
```

Reply:
Not sure which version is actually better/morer readable, so feel free to leave things as is if you deem so preferable.

Reply:
I used the first method because testing showed it's microseconds faster. :D

I'll use yours, it's more readable/understandable for the masses.